### DISCUSSION TITLE
React Native - Robinhood Wallet doesn't connect

### DISCUSSION THREAD

**Discussion by fandreirebaseit** [WalletConnect/walletconnect-monorepo#3630] [Developer Support]
Hi, 

Robinhood wallet isn't connecting via wallet connect v2. The behaviour is that wallet connect opens the robinhood wallet and after I press connect in robinhood wallet nothing happens, it doesn't go back in the app and connect the wallet, the rest of the wallets are connecting fine.

Thanks

**Comment by glitch-txs** [comment]
hi is this iOS or android?

**Reply by fandreirebaseit** [reply]
the problem is on both platforms, as we are developing using react native and migrated from v1 to v2 and right now using @walletconnect/modal-react-native

**Comment by Chris-Imade** [comment]
Hi, how can this issue be replicated?

**Reply by fandreirebaseit** [reply]
on your mobile app try to connect using WalletConnect the robinhood wallet
